apiserver/facades/controller/caasunitprovisioner/provisioner.go

```diff
@@ -218,15 +217,10 @@ func (a *Facade) UpdateApplicationsUnits(args params.UpdateApplicationUnitArgs)
 			continue
 		}
 
-		// Get the app config so we know how to handle the update.
-		// It depends on whether Juju or the cloud manage the units.
-		appConfig, err := app.ApplicationConfig()
-		if err != nil {
-			result.Results[i].Error = common.ServerError(err)
-			continue
-		}
-
-		if appConfig.GetBool(caas.JujuManagedUnits, caas.JujuDefaultJujuManagedUnits) {
+		// always set to `false` to disable `jujuManagedUnits`
+		jujuManagedUnits := false
+		if jujuManagedUnits {
```

Comment:
Personally, I would have liked to see here a warning for the users.

So rather than deleting the logic that obtained the value from user config, still get it and check if it's set, something like

``` appConfig.GetBool(caas.JujuManagedUnits)```

if that value is set, warn the user that we are ignoring their setting and deprecating it in future releases?...

Reply:
yes, agreed. it will be nicer for users. doing it now